src/plugins/data_source_management/public/components/create_data_source_wizard/create_data_source_wizard.tsx

```diff
@@ -3,15 +3,272 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import { EuiTitle } from '@elastic/eui';
-import React from 'react';
-import { withRouter } from 'react-router-dom';
+import {
+  EuiButton,
+  EuiFieldText,
+  EuiForm,
+  EuiFormRow,
+  EuiGlobalToastList,
+  EuiGlobalToastListToast,
+  EuiHorizontalRule,
+  EuiPageContent,
+  EuiPanel,
+  EuiSpacer,
+  EuiText,
+} from '@elastic/eui';
+import React, { useState } from 'react';
+import { RouteComponentProps, withRouter } from 'react-router-dom';
+import { i18n } from '@osd/i18n';
+import { useEffectOnce } from 'react-use';
+import { FormattedMessage } from '@osd/i18n/react';
+import { useOpenSearchDashboards } from '../../../../opensearch_dashboards_react/public';
+// eslint-disable-next-line @osd/eslint/no-restricted-paths
+import { DataSourceSavedObject } from '../../../../data_source/public/types';
+import { DataSourceCreationConfig } from '../../service';
+import { SavedObjectFinderUi } from '../../../../saved_objects/public';
+import { Header } from './components/header';
+import { DataSourceManagementContext } from '../../types';
+import { getCreateBreadcrumbs } from '../breadcrumbs';
+import { AuthenticationTabs, AuthenticationTabItem } from './components/authentication_tabs';
+import { CreateNewCredential } from './components/create_new_credential';
+
+interface SelectedSavedObj {
+  id: string;
+  type: string;
+  name?: string;
+}
+
+type CreateDataSourceWizardProps = RouteComponentProps;
+
+const CreateDataSourceWizard: React.FunctionComponent<CreateDataSourceWizardProps> = (
```

Comment:
we can comment out all logics around dataSource since I ask you to remove the dependency on datasource, I can later help with this creation piece of logic. For example: 
- fetchSavedDataSources();  
- handleSubmit
- ...

Reply:
Done